warehouse/query-core/src/main/java/datawave/query/jexl/visitors/QueryPropertyMarkerVisitor.java

```diff
@@ -57,42 +64,47 @@
                     ExceededValueThresholdMarkerJexlNode.class);
     // @formatter:on
     
-    protected Set<String> typeIdentifiers = new HashSet<>();
-    protected List<JexlNode> sourceNodes;
+    private final Set<String> whitelist;
+    private final Set<String> blacklist;
+    private List<JexlNode> sourceNodes;
     
-    private boolean identifierFound = false;
-    
-    protected Set<String> rejectedIdentifiers = new HashSet<>();
-    private boolean rejectedIdentifiersFound = false;
-    
-    private QueryPropertyMarkerVisitor() {}
+    private boolean whitelistedTypeFound = false;
+    private boolean blacklistedTypeFound = false;
```

Comment:
Consider using alternative terminology like allow and deny vs. what you have here.

Reply:
Sure, no problem.